zhaquirks/tuya/ts0601_electric_heating.py

```diff
@@ -82,6 +82,102 @@ def _update_attribute(self, attrid, value):
             self.endpoint.device.ui_bus.listener_event("child_lock_change", value)
 
 
+class AVATTOME81AHManufCluster(TuyaManufClusterAttributes):
+    """Manufacturer Specific Cluster of some electric heating thermostats."""
+
+    attributes = {
+        MOESBHT_TARGET_TEMP_ATTR: ("target_temperature", t.uint32_t, True),
+        MOESBHT_TEMPERATURE_ATTR: ("temperature", t.uint32_t, True),
+        MOESBHT_SCHEDULE_MODE_ATTR: ("schedule_mode", t.uint8_t, True),
+        MOESBHT_MANUAL_MODE_ATTR: ("manual_mode", t.uint8_t, True),
+        MOESBHT_ENABLED_ATTR: ("enabled", t.uint8_t, True),
+        MOESBHT_RUNNING_MODE_ATTR: ("running_mode", t.uint8_t, True),
+        MOESBHT_CHILD_LOCK_ATTR: ("child_lock", t.uint8_t, True),
+    }
+
+    def _update_attribute(self, attrid, value):
+        super()._update_attribute(attrid, value)
+        if attrid == MOESBHT_TARGET_TEMP_ATTR:
+            self.endpoint.device.thermostat_bus.listener_event(
+                "temperature_change",
+                "occupied_heating_setpoint",
+                value * 100,  # degree to centidegree
+            )
+        elif attrid == MOESBHT_TEMPERATURE_ATTR:
+            self.endpoint.device.thermostat_bus.listener_event(
+                "temperature_change",
+                "local_temperature",
+                value * 100,  # degree to centidegree
+            )
+        elif attrid == MOESBHT_SCHEDULE_MODE_ATTR:
+            if value == 0:  # value is inverted
+                self.endpoint.device.thermostat_bus.listener_event(
+                    "program_change", "scheduled"
+                )
+        elif attrid == MOESBHT_MANUAL_MODE_ATTR:
+            if value == 0:  # value is inverted
+                self.endpoint.device.thermostat_bus.listener_event(
+                    "program_change", "manual"
+                )
+        elif attrid == MOESBHT_ENABLED_ATTR:
+            self.endpoint.device.thermostat_bus.listener_event("enabled_change", value)
+        elif attrid == MOESBHT_RUNNING_MODE_ATTR:
+            # value is inverted
+            self.endpoint.device.thermostat_bus.listener_event(
+                "state_change", 1 - value
+            )
+        elif attrid == MOESBHT_CHILD_LOCK_ATTR:
+            self.endpoint.device.ui_bus.listener_event("child_lock_change", value)
+
+
+class BeokTGR85_ZBManufCluster(TuyaManufClusterAttributes):
+    """Manufacturer Specific Cluster of some electric heating thermostats."""
+
+    attributes = {
+        MOESBHT_TARGET_TEMP_ATTR: ("target_temperature", t.uint32_t, True),
+        MOESBHT_TEMPERATURE_ATTR: ("temperature", t.uint32_t, True),
+        MOESBHT_SCHEDULE_MODE_ATTR: ("schedule_mode", t.uint8_t, True),
+        MOESBHT_MANUAL_MODE_ATTR: ("manual_mode", t.uint8_t, True),
+        MOESBHT_ENABLED_ATTR: ("enabled", t.uint8_t, True),
+        MOESBHT_RUNNING_MODE_ATTR: ("running_mode", t.uint8_t, True),
+        MOESBHT_CHILD_LOCK_ATTR: ("child_lock", t.uint8_t, True),
+    }
+
```

Comment:
If the no device response to commands is related to the `NO_MANUFACTURER_ID` that tuya is recently using, maybe this can be fixed overwritting the `write_attributes` method like (not tested):
```python
    async def write_attributes(self, attributes, manufacturer=None):
        """Overwrite to force manufacturer code."""

        return await super().write_attributes(
            attributes, manufacturer=foundation.ZCLHeader.NO_MANUFACTURER_ID
        )

```

You will need also to add another import:
```python
from zigpy.zcl import foundation
```